apps/condo/domains/contact/components/contactRoles/BaseContactRoleForm.tsx

```diff
@@ -47,13 +46,12 @@ export const BaseContactRoleForm: React.FC<BaseTicketPropertyHintFormProps> = ({
 
     const router = useRouter()
 
-    const createContactRoleAction = ContactRole.useCreate({})
-    const softDeleteContactRoleAction = ContactRole.useSoftDelete()
-
     const validationRules = useContactRoleValidationRules()
 
     const handleFormSubmit = useCallback(async (values) => {
         const initialContactRoleId = get(initialValues, 'id')
+        const normalizedName = values.name && values.name.trim()
```

Comment:
Name can be undefined here?

Reply:
It seems not, because field must be filled

fixed